README.md

```diff
@@ -82,34 +82,37 @@ Week 02 _(~1.5 hours)_
 
 - Linting
 - Continuous integration
+
+Week 03 _(~2.5 hours)_
+
 - Assignment #1
 
-Week 03 _(~1.5 hours)_
+Week 04 _(~1.5 hours)_
 
 - Git strategies
 - Code reviews
 - Assignment #2
 
-Week 04 _(~3 hours)_
+Week 05 _(~3 hours)_
 
 - Testing
 - Project structure
 - Text editors
 
-Week 05 _(~2 hours)_
+Week 06 _(~2 hours)_
 
 - Assignment #3
 
-Week 06 _(~2.5 hours)_
+Week 07 _(~2.5 hours)_
 
 - Object-oriented programming
 
-Week 07 _(~2.5 hours)_
+Week 08 _(~2.5 hours)_
 
 - Design patterns
 - Packaging and publishing
 
-Week 08 _(~2 hours)_
+Week 09 _(~2 hours)_
```

Comment:
9 weeks is so long. I'm thinking that we could add another calendar that is an accelerated one for people that have 10-15 hours per week for study. Might be nice to let people know that it can be done in 3 or 4 weeks if you've got more time. I think this might be worth it for the Alfas so maybe we bring it up with Diogo.

Reply:
Since this repo is growing on a separate track from NOS, we can reunify some weeks to make it fit into an 8-week schedule again. Do you have any suggestions?